src/components/Calculator.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import { Col, Row, Button } from 'react-bootstrap';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
+        <div className="container mx-auto pt-5">
+          <Row className="row d-flex justify-content-center py-5">
+            <Col>
+              <h1>Magic Calculator</h1>
+            </Col>
+          </Row>
+          <Row className="row d-flex justify-content-center">
+            <Col xs={12} className="w-100">
+              <Button
+                variant="btn btn-secondary col-8 border-dark text-end pe-3 py-3"
+              >
+                0
```

Comment:
- [ ] Please, It would be better if you follow the requirements about Calculator's design (you should use the same color, background color):

![calculator-design](https://user-images.githubusercontent.com/79876959/143376411-86f1900c-ca67-491a-9634-772b1a241a18.PNG)